pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -924,9 +944,9 @@ func (r *ReconcileVirtualMachineImport) updateProgress(instance *v2vv1alpha1.Vir
 	return nil
 }
 
-func (r *ReconcileVirtualMachineImport) afterSuccess(vmName types.NamespacedName, vmiName types.NamespacedName, p provider.Provider) error {
+func (r *ReconcileVirtualMachineImport) afterSuccess(vmName types.NamespacedName, vmiName types.NamespacedName, p provider.Provider, instance *v2vv1alpha1.VirtualMachineImport) error {
```

Comment:
The same data is passed twice as parameter in a different format:

`instanceNamespacedName := types.NamespacedName{Name: instance.Name, Namespace: instance.Namespace}`

The second parameter `vmiName` can be omitted and calculated and be initialized within the method.

Reply:
:+1: